DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/HttpClientExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace GitHubExtension.Templates.ExtensionMethods
+{
+    public static class HttpClientExtension
+    {
+        public static async Task<HttpResponseMessage> GetResponse(this HttpClient httpClient, HttpRequestMessage httpRequestMessage)
+        {
+            var _httpClient = new HttpClient();
```

Comment:
It's not a good idea place initialization HttpClient inside extension method. Can you explain why?